core/modules/layout/plugins/styles/layout_style_region.inc

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * @file
+ * Class providing forms and settings for the Layout region style.
+ */
+class LayoutStyleRegion extends LayoutStyle {
+  /**
+   * Indicator as to whether this is a style on the region or on a block.
+   *
+   * @var boolean
+   */
+  var $is_region = TRUE;
```

Comment:
Oh, I see we already have checking for `is_region` in the `LayoutStyle` class. Nice.